team-policy.md

```diff
@@ -0,0 +1,144 @@
+# Style Team Policy Document
+
+This is a living document tracking the active policies of the style team. It is intended to fill a similar role to the books that many[^1] teams[^2] maintain[^3] independently[^4]. If in the future Rust establishes central linking/indexing/aggregating of policies, these will need to appear there.
+
+* **Original RFC**: https://rust-lang.github.io/rfcs/3309-style-team.html
+* **Charter**: https://github.com/rust-lang/fmt-rfcs/blob/master/charter.md
+
+## Style Guide Evolution
+
+The Rust style guide will generally match the latest version of the Rust style; the style team does not plan to maintain multiple branches of the style guide for different editions, in part because formatting for new constructs will apply to any edition supporting those constructs.
+
+Whenever possible, style decisions should be made before a new construct is stabilized. However, style decisions shall not be considered a blocker for stabilization.
+
+## Operational Roles
+
+The style team has various operational roles to delegate related and ongoing work to specific individuals, to clarify on our operations, and to coordinate sharing feedback with the members in those roles to enable iterative self improvement.
+
+### Operational Lead
+
+* Current Holder: [@calebcartwright](https://github.com/calebcartwright)
+* Next Feedback Date: TODO
+
+In order to stay in touch with where we want to be heading in the future, we need leadership. A team leader is paying attention to the team’s operations in relation to the team’s aim. What needs to be done, who agreed to do it. What is in the future to decide?
+
```

Comment:
Just ran into this article and thought it might be a good one to save as recommended reading for anyone who takes on this role:

```suggestion

#### Recommended Reading

* https://www.sociocracyforall.org/leadership-without-hierarchy/

```